units/provisioning_create_host.go

```diff
@@ -353,11 +353,25 @@ func (j *createHostJob) isImageBuilt(ctx context.Context) (bool, error) {
 		return false, errors.Errorf("parent for host '%s' not running", j.host.Id)
 	}
 	if ok := parent.ContainerImages[j.host.DockerOptions.Image]; ok {
+		grip.Info(message.Fields{
```

Comment:
do we really need this logging?

Reply:
I find it useful. It should only log once per docker image, so I'd like to keep it in